lib/main.dart

```diff
@@ -259,15 +260,19 @@ class _MyHomePageState extends State<MyHomePage> {
                     : null,
                 child: Chip(
                   avatar: icon,
-                  label: Text(slot.item.name),
+                  label: Text(name),
                 ));
           });
     }
 
+    final indexOfSpace = name.indexOf(" ");
+
     return Card(
       child: ListTile(
         title: Row(children: [
-          Text(slot.item.name),
+          Text((name.length <= 15 || indexOfSpace == -1)
```

Comment:
Different screen sizes will change this! Try to find a way to actually do text wrapping in Flutter